ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift

```diff
@@ -1,10 +1,18 @@
-//
-//  ENCoreDelegate.swift
-//  ErnRunner
-//
-//  Created by Lianci Liu on 6/27/19.
-//  Copyright © 2019 Walmart. All rights reserved.
-//
+/*
+ * Copyright 2019 Walmart Labs
+ 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ 
```

Comment:
These empty lines need a `*` too (a space followed by `*`). Otherwise it's a whitespace error (trailing spaces).
Please change to
```
/*
 * Copyright 2019 Walmart Labs
 *
 * Licensed under ...
 * ...
```
Otherwise looks good! 👍